tests/e2e/cucumber/features/file-action/download.feature

```diff
@@ -62,3 +62,48 @@ Feature: Download
       | resource       | type |
       | testavatar.jpg | file |
     And "Brian" logs out
+
+  @posix
+  Scenario: download large archive
+    Given the user creates a file "largefile500Mb.txt" of "500MB" size in the temp upload directory
+    And the user creates a file "largefile400Mb.txt" of "400MB" size in the temp upload directory
+    And the user creates a file "largefile300Mb.txt" of "300MB" size in the temp upload directory
+    And "Alice" uploads the following local file into personal space using API
+      | localFile                     | to             |
+      | temp/largefile500Mb.txt | largefile500Mb.txt |
+      | temp/largefile400Mb.txt | largefile400Mb.txt |
+      | temp/largefile300Mb.txt | largefile300Mb.txt |
+    When "Alice" logs in
+    And "Alice" downloads the following resources using the batch action
+      | resource          | type |
+      | largefile500Mb.txt | file |
+      | largefile400Mb.txt | file |
+    When "Alice" selects all files
+    Then the download button should be disabled for user "Alice" with the tooltip:
+      """
+      The selection exceeds the allowed archive size (max. 1.1 GB)
+      """
+    And "Alice" logs out
+
+  
+  @posix
+  Scenario: download large archive using upload with posix 
+    Given "Alice" creates the following files into personal space using API
+      | pathToFile | content     |
+      | new.txt | new content |
+    And "Alice" creates following filess using CLI:
+      | resource         | size |
+      | testfile500Mb.bin| 500  |
+      | testfile400Mb.bin| 400  |
+      | testfile300Mb.bin| 300  |
+    When "Alice" logs in
+    And "Alice" downloads the following resources using the batch action
```

Comment:
There is no "Then" step for this action. How do we know the download was successful?

Reply:
Yeah, I know. we cannot check that file was download on UI. we check that file was downloaded as archiver `expect(name).toBe('download')` inside the method